main.tex

```diff
@@ -1,9 +1,13 @@
 \documentclass[english]{notes}
 
-\usepackage{./mama}
+
 \usepackage{blkarray}
 \usepackage{wrapfig}
 \usepackage{graphicx}
+\usepackage{subfiles}%Allows compilation of subfiles.
+\usepackage{./mama}%keep order of mama and subfiles packages
+
+\tikzexternaldisable %Solves compatibility problems. In order to obtain faster speed in compilation try to remove it. 
```

Comment:
You can remove `tikzexternalize` altogether if we use subfiles, by editing notes.cls

Reply:
You prefer I remove your lines from the package notes? If I keep tikzexternaldisable in main it is easier to comment it whenever you want.